src/main/resources/assets/nutrition/configs/nutrients/vegetable.json

```diff
@@ -753,6 +753,9 @@
 			"pvj:cactus_salad",
 			"pvj:mystical_potato",
 			"quark:root",
+			"quark:root_flower",
```

Comment:
Could you mark this one as `"quark:root_flower:0"`?  It's better to be explicit when using metadata like this.

Otherwise looks good!